src/applications/mhv/secure-messaging/tests/containers/ThreadDetails.unit.spec.jsx

```diff
@@ -185,6 +185,12 @@ describe('Thread Details container', () => {
 
     const state = {
       sm: {
+        folders: {
+          folder: {
+            folderId: 0,
+            name: 'Inbox',
+          },
```

Comment:
We should have these objects in mocks in `/fixtures` that we can re-use here and in other tests too